supplementary_style_guide/style_guidelines/support.adoc

```diff
@@ -0,0 +1,25 @@
+[[support]]
+= Support
+
+[discrete]
+== Technology Preview
+
+Technology Preview documentation guidelines apply to any form of early access feature that is not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic install and configuration information.
+
+* Add an admonition labeled IMPORTANT at the beginning of the Technology Preview content.
+* Use initial uppercase capitalization, that is, Technology Preview.
+* Include the Technology Preview feature in the Release Notes.
+* Maintain a list of features that are currently in Technology Preview status in the Release
+
+When a minor release becomes generally available, remove the IMPORTANT admonition from the Release Notes and any other document that includes content about the feature.
+
+Use the following template. Replace <feature> with the feature name:
```

Comment:
```suggestion
Use the following template. Replace _<feature_name>_ with the feature name:
```
Per SSG: italicize user-replaced values. Also consider using the format `feature_name` to be consistent with the SSG formatting guidance. Thanks!